ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>Cloud SDK -&gt; Google Cloud SDK</p><p> </p><p>run <code>pipe ... in a shell -&gt; run the command </code>pip  ...` on the command-line in a terminal shell</p><p> </p><p>same for run jupyter notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It's just "Cloud SDK"</p><p> </p>

Reply:
<p>Otherwise, fixed</p>